tests/e2e-pw/tests/product-gallery/inner-blocks/product-gallery-thumbnails/product-gallery-thumbnails.block_theme.spec.ts

```diff
@@ -0,0 +1,374 @@
+/**
+ * External dependencies
+ */
+import { test, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import { addBlock } from './utils';
+
+const blockData = {
+	name: 'woocommerce/product-gallery-thumbnails',
+	mainClass: '.wp-block-woocommerce-product-gallery-thumbnails',
+	selectors: {
+		frontend: {},
+		editor: {
+			thumbnails: '.wp-block-woocommerce-product-gallery-thumbnails',
+			noThumbnailsOption: 'button[data-value=off]',
+			leftPositionThumbnailsOption: 'button[data-value=left]',
+			bottomPositionThumbnailsOption: 'button[data-value=bottom]',
+			rightPositionThumbnailsOption: 'button[data-value=right]',
+		},
+	},
+	slug: 'single-product',
+	productPage: '/product/v-neck-t-shirt/',
+};
+
+test.describe( `${ blockData.name }`, () => {
+	test.beforeEach( async ( { requestUtils, admin, editorUtils } ) => {
```

Comment:
Do we need to delete all templates before **and** after each test? Keeping just one of these deletions (either `beforeEach` or `afterEach`) should be enough, no?

Also, could we explore deleting all templates just once, by relying on `beforeAll`, instead of on each test individually?

Reply:
Awesome, thanks! 🙌